airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/config/AirbyteDestinationConfig.java

```diff
@@ -0,0 +1,83 @@
+package io.airbyte.cdk.db;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.airbyte.cdk.db.AirbyteSourceConfig.SourceConfigBuilder;
+import io.airbyte.commons.io.IOs;
+import io.airbyte.commons.json.Jsons;
+import java.nio.file.Path;
+import java.util.Map;
+
+public class AirbyteDestinationConfig extends AirbyteConfig<AirbyteDestinationConfig> {
+  public static class DestinationConfigBuilder extends AirbyteConfigBuilder<AirbyteDestinationConfig> {
```

Comment:
It's OK for inner classes to have short names, as they're rarely imported directly.